tests/stdlib/tnetdial.nim

```diff
@@ -2,7 +2,6 @@ discard """
   cmd: "nim c --threads:on $file"
   exitcode: 0
   output: "OK"
-  disabled: "travis"
```

Comment:
Even after this fix, this test fails on Travis.

Reply:
Why does it fail? What's the problem? Timing sensitivity?